libvcx/src/aries/messages/connection/did_doc.rs

```diff
@@ -20,7 +20,8 @@ pub struct DidDoc {
     pub id: String,
     #[serde(default)]
     #[serde(rename = "publicKey")]
-    pub public_key: Vec<Ed25519PublicKey>, // TODO: A DID document MAY include a publicKey property??? (https://w3c.github.io/did-core/#public-keys)
+    pub public_key: Vec<Ed25519PublicKey>,
+    // TODO: A DID document MAY include a publicKey property??? (https://w3c.github.io/did-core/#public-keys)
```

Comment:
Why are you adding another line?

Reply:
it was my reformatter, personally I am indifferent; reverted on single line.